.circleci/config.yml

```diff
@@ -514,6 +514,10 @@ jobs:
       - checkout
       - restore_dependency_cache
       - gh-config
```

Comment:
Taken from the build on windows job for using the latest version of npm.  I'm not able to test without hoop jumping locally, but will react to if this doesn't work as expected in circle.